examples/incremental_learning/helmet-detection-modelbox/incremental_learning-modelbox/test/mock/mock_modelbox.cc

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
It is suggestted to add user guide to introduce what is the function of each file like this.